src/test/scala/com/databricks/sparkdl/ImageUtilsSuite.scala

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2017 Databricks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.databricks.sparkdl
+
+import java.awt.Color
+import java.io.File
+import javax.imageio.ImageIO
+
+import scala.util.Random
+import org.apache.spark.image.ImageSchema
+import org.apache.spark.sql.Row
+import org.scalatest.FunSuite
+
+object ImageUtilsSuite {
+  val biggerImage: Row = {
+    val biggerFile = getClass.getResource("/images/00081101.jpg").getFile
```

Comment:
can we put that in a subdirectory such as `/images/sparkdl`? Remember that all the namespaces of resources gets flattened into a single namespace within the JVM -> risk of collisions.